### ISSUE TITLE
Account recovery (or deletion) request (**TestPyPI**)

### ISSUE THREAD

**Issue by andthum** [pypi/support#2573]
### PyPI Username

`andthum` (on **TestPyPI**)

### Reason for request

Lost 2FA authentication app or security token.

This only applies to my account on [**TestPyPI**](https://test.pypi.org/)! I still have access to my account with the same username on the "real" PyPI page.

I thought accounts on TestPyPI get deleted on a regular basis, so I was not so careful about it.  But now I wanted to upload a package called `mdtools` to TestPyPI and had to realize that I already uploaded a package with the same name more than three years ago.  So, the upload was rejected because a package with the same name already exists.

So long story short: I do not really need access to my account on TestPyPI.  I just need the `mdtools` package I uploaded previously to get removed.  So, could you please either grant me access to my account on TestPyPI or remove the account with all its projects (there should only be the mdtools project)?  Many thanks!

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by ewdurbin** [comment]
2FA disabled and password reset for `andthum` on TestPyPI, since the user/email matches to PyPI.
